rust/lance-core/src/format/page_table.rs

```diff
@@ -177,22 +175,22 @@ mod tests {
         page_table.set(12, 2, page_info.clone());
         page_table.set(12, 3, page_info.clone());
 
-        let path = Path::from("test");
-        let object_store = ObjectStore::memory();
+        let test_dir = tempfile::tempdir().unwrap();
+        let path = test_dir.path().join("test");
```

Comment:
Why switch away from working in memory? Seems to make sense for unit tests.

Reply:
Because it introduces dependency of our `ObjectStore`, which takes more time to migrate.